demos/projects/ESPRESSIF/aziotkit/components/azure-iot-kit-sensors/src/oled.h

```diff
@@ -0,0 +1,32 @@
+/* OLED screen Example
+
+   For other examples please check:
+   https://github.com/espressif/esp-iot-solution/tree/master/examples
+
+   This example code is in the Public Domain (or CC0 licensed, at your option.)
+
+   Unless required by applicable law or agreed to in writing, this
+   software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied.
+ */
+#ifndef _APP_OLED_H_

```

Comment:
can we name it  based on file name? and for consistency in similar format to other file.